README.md

```
@@ -491,7 +491,112 @@ The command generates evaluation metrics and visualization plots:
+      model.transformer.d_model: 512
+      model.transformer.n_layers: 3
+      training.gradient_clip_val: 0.5
+```
+
+- All keys must be valid params listed under `sweep.params`
+- Multiple starting points are supported — each runs as a separate trial
+- Ignored when `sweep.resume=true` (TPE already has prior knowledge)
+
+### Output Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)